examples/jsm/controls/OrbitControls.js

```diff
@@ -883,26 +883,20 @@ class OrbitControls extends EventDispatcher {
 
 		function onPointerUp( event ) {
 
-		    removePointer( event );
-
-		    if ( pointers.length === 0 ) {
-
-		        scope.domElement.releasePointerCapture( event.pointerId );
-
-		        scope.domElement.removeEventListener( 'pointermove', onPointerMove );
-		        scope.domElement.removeEventListener( 'pointerup', onPointerUp );
+			removePointer( event );
 
-		    }
+			if ( pointers.length === 0 ) {
 
-		    scope.dispatchEvent( _endEvent );
+				scope.domElement.releasePointerCapture( event.pointerId );
 
-		    state = STATE.NONE;
+				scope.domElement.removeEventListener( 'pointermove', onPointerMove );
+				scope.domElement.removeEventListener( 'pointerup', onPointerUp );
```

Comment:
Shouldn't `pointercancel` listener also be removed here?

Reply:
`pointermove` and `pointerup` are added differently and thus have to be removed in a different way, too.